keystrokes/src/components/Home/Home.js

```diff
@@ -15,9 +15,13 @@ function Home(props) {
 	});
 
 	async function collectUserInfo() {
-		const snapShot = await ref.child(userPath).once('value');
-		if (snapShot.exists()) {
-			return snapShot.val();
+		try {
+			const snapShot = await ref.child(userPath).once('value');
```

Comment:
You might want to make a reference to a specific path e.g. 
userRef = ref.child(`Users/${userId}`)

and then reference it later userRef.once userRef, etc..